modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1361,16 +1437,19 @@ This command does not take any options, except for the default ones: `--help`, `
 ----
 This command describes the history of the alarm passed in `<name>`.
 
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+// CLI DEFAULTS
+include::partial$cli-default-options.adoc[tag=CLIdefaultOptions]
 
 === runtime-mgr cloudhub-application list
 
 ----
 > runtime-mgr cloudhub-application list [options]
 ----
 
-This command lists all applications available in your Anypoint Platform CLI. It returns your application name, its status, the number of vCores assigned and the last time it was updated. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command lists all applications available in your Anypoint Platform CLI. It returns your application name, its status, the number of vCores assigned and the last time it was updated. 
```

Comment:
do we use Oxford comma here? "... assigned, and the..."

Reply:
Yes, but I'm not editing that text right now.